lib/taskana-core/src/main/java/pro/taskana/classification/api/ClassificationService.java

```diff
@@ -116,18 +144,21 @@ Classification updateClassification(Classification classification)
   /**
    * This method provides a query builder for querying the database.
    *
-   * @return a {@link ClassificationQuery}
+   * @return a {@linkplain ClassificationQuery}
    */
   ClassificationQuery createClassificationQuery();
 
   /**
-   * Creating a new {@link Classification} with unchangeable default values. It will be only
-   * generated and is not inserted until CREATE-call.
+   * Creating a new {@linkplain Classification} with unchangeable default values. It will be only
+   * generated and is not inserted until {@linkplain
+   * ClassificationService#createClassification(Classification)} call.
    *
-   * @param key the key of the classification
-   * @param domain the domain of the new classification
-   * @param type the type of the new classification
-   * @return classification to specify
+   * @param key the {@linkplain Classification#getKey() key} of the {@linkplain Classification}
+   * @param domain the {@linkplain Classification#getDomain() domain} of the new {@linkplain
+   *     Classification}
+   * @param type the {@linkplain Classification#getType() type} of the new {@linkplain
+   *     Classification}
+   * @return a default {@linkplain Classification}
```

Comment:
Do we return a "default" classification or rather a "initialized" classification?

Reply:
"initialized" sounds better. We don't set any values other than the arguments of the method.